README.md

```diff
@@ -184,5 +170,8 @@ This way the scheduler will take all resources allocated to the role and make su
 
 It is always recommended to run the scheduler with a Mesos role and reserved resources in such cases to make sure that scheduler is being offered resources for all nodes in the cluster.
 
-### Framework shutdown
-The scheduler can survive `SIGKILL` or being lost (system crashes etc.). If you want to completely de-register the framework and shutdown all tasks, just stop the scheduler with a plain `SIGTERM`.
+# Framework shutdown
+The tasks will restart if they receive a `SIGKILL` or are lost (system crashes etc.). If you want to completely de-register the framework and shutdown all tasks, just stop the scheduler with a plain `SIGTERM`.
```

Comment:
I'm afraid this is wrong? Nothing will happen when scheduler is being killed (with `SIGKILL`)